src/icons/revert.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 21 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Same icon was added it in the transaction list with a different name, you can reuse it.